core/src/kura.rs

```diff
@@ -51,21 +51,23 @@ impl Kura {
     /// to access the block store indicated by the provided
     /// path.
     pub fn new(
-        mode: Mode,
-        block_store_path: &Path,
-        debug_output_new_blocks: bool,
+        config: &Configuration,
+        // mode: Mode,
```

Comment:
These commented-out arguments can probably be removed

Reply:
Removed